packages/db/lib/schema.js

```diff
@@ -86,6 +86,9 @@ const db = {
           schemaPath: {
             type: 'string',
             resolvePath: true
+          },
+          enabled: {
+            type: 'boolean'
```

Comment:
It's better if those also allow strings so their values can be filled by env variables, check out https://github.com/platformatic/platformatic/blob/3f4281f3b8c6f480d76b4f854523c6444b2f2f0e/packages/service/lib/schema.js#L115 to see how to do it.